e2e/connection_test.go

```diff
@@ -114,7 +114,7 @@ func consumerConnectFlow(t *testing.T, tequilapi *tequilapi_client.Client, consu
 	})
 	assert.NoError(t, err)
 
-	connectionStatus, err = tequilapi.Connect(consumerID, proposal.ProviderID, endpoints.ConnectOptions{true})
+	connectionStatus, err = tequilapi.Connect(consumerID, proposal.ProviderID, "openvpn", endpoints.ConnectOptions{true})
```

Comment:
Shouldn't we pass here a proposal too?

Reply:
out of scope, we'll need to fix the api part though, to avoid having to query for the service type in general. I've created #549 for this.

Reply:
Awesome 👍